src/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,43 @@
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+
+export const Navigation = () => {
+  return (
+    <nav
+      className="navbar is-light is-fixed-top is-mobile has-shadow"
+      data-cy="Nav"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            className={({ isActive }) => (
+              cn(
+                'navbar-item',
+                {
+                  'is-active': isActive,
+                },
+              )
+            )}
```

Comment:
it takes way too much of space. I can't promise my suggestion will work exactly this way because of those braces, but I believe you got the main idea 😄 
```suggestion
            className={({ isActive }) => cn(
                'navbar-item', {
                  'is-active': isActive,
               })}
```